rca/people/models.py

```diff
@@ -668,7 +677,19 @@ def clean(self):
             raise ValidationError(errors)
 
 
-class StudentPage(PerUserPageMixin, BasePage):
+def student_base_page_content_panels(content_panels):
+    for panel in content_panels:
+        if isinstance(panel, FieldPanel) and panel.field_name == "title":
+            """Developer note:
+            This will only loop over the top level panels which is OK currently
+            because there's only one field panel with the name `title` at the moment
+            which originates from the Wagtail Page model.
+            If BasePage.content_panels is implemented then this will need to be updated"""
+            panel.permission = "superuser"
+    return content_panels
```

Comment:
Thats a good description.

So here it's just looping whats coming from BasePage.content_panels and if it has the title field then make it
`FieldPanel("title", permission="superuser")`

I suppose it could be defined in content_panels on the BasePage but then it would be permission="superuser" all over the admin.

Reply:
yea though as much :+1: